src/test/java/test/factory/Factory2Sample.java

```diff
@@ -41,10 +41,10 @@ public void tearDown() {
           "testInputImages",
           "tearDown",
           "setUp",
-          "testInputImages",
+          "testImages",
```

Comment:
This is one of the test changes that I am least confident is necessary. If you think this a test that shouldn't be changed like this, I can dig deeper for a solution that will preserve what this test is looking for while satisfying my new requirements.